eegnb/experiments/auditory_ssaep/ssaep_onefreq.py

```diff
@@ -0,0 +1,152 @@
+"""
+Generate Steady-State Auditory Evoked Potential (SSAEP)
+=======================================================
+
+Steady-State Auditory Evoked Potential (SSAEP) - also known as Auditory
+Steady-State Response (ASSR) - stimulus presentation.
+
+This variant allows for a single tone (single pair of carrier and modulation frequency)
+
+"""
+
+from time import time, sleep
+from optparse import OptionParser
+
+import numpy as np
+from pandas import DataFrame
+from psychopy import prefs
+prefs.general['audioLib'] = ['pygame']
+from psychopy import visual, core, event, sound
+from pylsl import StreamInfo, StreamOutlet
+from scipy import stats
+
+import os
+from glob import glob
+from random import choice
+
+
+from eegnb import generate_save_fn
+
+
+def present(duration=120, eeg=None, save_fn=None, iti = 0.5, soa = 3.0, jitter = 0.2, 
+            n_trials = 150, cf1 = 1000, amf1 = 40):
+
+    # Create markers stream outlet
+    info = StreamInfo('Markers', 'Markers', 1, 0, 'int32', 'myuidw43536')
+    outlet = StreamOutlet(info)
+
+    markernames = [1]
+    start = time()
+
+    # Set up trial parameters
+    record_duration = np.float32(duration)
+
+    # Set up trial list
+    stim_freq = np.zeros((n_trials,), dtype=int)
+    trials = DataFrame(dict(stim_freq=stim_freq, timestamp=np.zeros(n_trials)))
+
+    # Setup graphics
+    mywin = visual.Window([1920, 1080], monitor='testMonitor', units='deg',
+                          fullscr=True)
+    fixation = visual.GratingStim(win=mywin, size=0.2, pos=[0, 0], sf=0,
+                                  rgb=[1, 0, 0])
+    fixation.setAutoDraw(True)
+
+
+    def generate_am_waveform(carrier_freq, am_freq, secs=1, sample_rate=44100,
```

Comment:
I just realized that `secs` here is defined as 1, and has a different value than `soa`, which is defined at the beginning. The value I would need now is 1, but maybe better change it when calling the experiment.